cmd/influx/query.go

```diff
@@ -101,15 +103,64 @@ func fluxQueryF(cmd *cobra.Command, args []string) error {
 	}
 	u.RawQuery = params.Encode()
 
-	body, _ := json.Marshal(map[string]interface{}{
+	var body_map = map[string]interface{}{
 		"query": q,
 		"type":  "flux",
 		"dialect": map[string]interface{}{
 			"annotations": []string{"group", "datatype", "default"},
 			"delimiter":   ",",
 			"header":      true,
 		},
-	})
+	}
+
+	if queryFlags.profiler {
+		body_map["extern"] = map[string]interface{}{
```

Comment:
Can you move the `map[string]interface{}{...}` definition to a top-level `const`? IMO it'd make it more clear that it's a "magic" constant instead of something that depends on user input.

Reply:
Once it's moved, is there a docs page we could link to in a comment that explains the structure? In case something changes in the future and we need to figure out how this should be updated to match